electron/package.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
"Dev" package.json for the Electron app. Only contains dependencies needed to **work** on the app, but nothing that the app needs to actually run.